test/e2e/tests/send-hex-address.spec.js

```diff
@@ -175,6 +179,7 @@ describe('Send ERC20 to a 40 character hexadecimal address', function () {
           windowHandles,
         );
         await driver.clickElement({ text: 'Add Token', tag: 'button' });
+        await driver.delay(veryLargeDelayMs * 4);
```

Comment:
Peter is going to push a fix to mock the API request that was causing the stall here and I've opened[ a PR in the controllers repo to timeout this request faster](https://github.com/MetaMask/controllers/pull/875) - since it appears to stall endlessly, when the token address being queried for is unknown to the API.